src/applications/vaos/utils/eligibility.js

```diff
@@ -176,16 +190,12 @@ export function getEligibilityChecks(eligibilityData) {
   if (!eligibilityChecks.directFailed) {
     eligibilityChecks = {
       ...eligibilityChecks,
-      directPastVisit:
-        directSchedulingEnabled &&
-        hasVisitedInPastMonthsDirect(eligibilityData),
-      directPastVisitValue:
-        directSchedulingEnabled &&
-        eligibilityData.directPastVisit.durationInMonths,
+      directPastVisit: hasVisitedInPastMonthsDirect(eligibilityData),
+      directPastVisitValue: eligibilityData.directPastVisit
+        ? eligibilityData.directPastVisit.durationInMonths
+        : null,
```

Comment:
I think this and `requestPastVisitValue` above could possibly be simplified with: `directPastVisitValue: eligibilityData.directPastVisit?.durationInMonths || null`